airbrakes/airbrakes.py

```diff
@@ -92,7 +92,8 @@ def update(self) -> None:
         i = 0
         for data_packet in data_packets:
             logged_data_packet = LoggedDataPacket(
-                state=self.state.name[0], extension=self.current_extension.value, timestamp=data_packet.timestamp
```

Comment:
where are you setting timestamp? Because timestamp has no initial value, I thought it has to be set upon creation

Reply:
`set_imu_data_packet_attributes` automatically does this, there is no need of duplication here. I will add a specific test for this in another PR.

Reply:
ugh, you might be right. So much for trying to separate my PRs.